templates/zerver/emails/confirm_new_email.html

```diff
@@ -1,5 +1,9 @@
 {% extends "zerver/emails/email_base_default.html" %}
 
+{% block illustration %}
+<img src="{{ email_images_base_uri }}email_logo.png"/>
```

Comment:
This is my fault, but could you modify this commit to remove the last `/` from `email_images_base_uri` and add them in here?